packages/codegen-openapi/README.md

```diff
@@ -0,0 +1,62 @@
+# 🪄 Hyper Fetch Openapi Request Generator 
+
+<p>
+  <a href="https://bettertyped.com/">
+    <img src="https://custom-icon-badges.demolab.com/static/v1?label=&message=BetterTyped&color=333&logo=BT" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch">
+    <img src="https://custom-icon-badges.demolab.com/github/stars/BetterTyped/hyper-fetch?logo=star&color=118ab2" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch/blob/main/License.md">
+    <img src="https://custom-icon-badges.demolab.com/github/license/BetterTyped/hyper-fetch?logo=law&color=yellow" />
+  </a>
+  <a href="https://www.npmjs.com/package/@hyper-fetch/graphql">
+    <img src="https://custom-icon-badges.demolab.com/npm/v/@hyper-fetch/graphql.svg?logo=npm&color=e76f51" />
+  </a>
+  <a href="https://api.codeclimate.com/v1/badges/eade9435e75ecea0c004/test_coverage">
+    <img src="https://api.codeclimate.com/v1/badges/eade9435e75ecea0c004/test_coverage" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch">
+    <img src="https://custom-icon-badges.demolab.com/badge/typescript-%23007ACC.svg?logo=typescript&logoColor=white" />
+  </a>
+  <a href="https://github.com/BetterTyped/hyper-fetch">
+    <img src="https://custom-icon-badges.demolab.com/badge/-Axios-E10098?logo=axios&logoColor=white" />
+  </a>
+  <a href="https://www.npmjs.com/package/@hyper-fetch/axios">
+    <img src="https://custom-icon-badges.demolab.com/bundlephobia/minzip/@hyper-fetch/axios?color=64BC4B&logo=package" />
+  </a>
+</p>
```

Comment:
Change badges urls to reflect new package `@hyper-fetch/axios` => `@hyper-fetch/codegen-openapi`. I see here the mix of the graphql and axios, so please check also the old packages - firebase, axios to make sure the badges are correct and pointing to the correct package urls.